sdk/confidentialledger/confidential-ledger-rest/README.md

```diff
@@ -164,6 +175,75 @@ const ledgerEntry: CreateLedgerEntryParameters = {
 const result = await client.path("/app/transactions").post(ledgerEntry);
 ```
 
+### Post Ledger Entry With CollectionId
+
+```ts snippet:ReadmeSamplePostLedgerEntryWithCollectionId
+import ConfidentialLedger, {
+  getLedgerIdentity,
+  LedgerEntry,
+  CreateLedgerEntryParameters,
+} from "@azure-rest/confidential-ledger";
+import { DefaultAzureCredential } from "@azure/identity";
+
+const { ledgerIdentityCertificate } = await getLedgerIdentity(
+  "test-ledger-name",
+  "https://identity.confidential-ledger.core.azure.com",
+);
+const credential = new DefaultAzureCredential();
+const client = ConfidentialLedger(
+  "https://test-ledger-name.confidential-ledger.azure.com",
+  ledgerIdentityCertificate,
+  credential,
+);
+// Type assertion is used to allow collectionId
+const entry = {
+  contents: "<content>",
+  collectionId: "my-collection",
+} as LedgerEntry & {
```

Comment:
Same question as below - how come there is a fix to `LedgerEntry` type to add `collectionId`? That seems like an omission on the `LedgerEntry` type that we should fix?